docs/DeveloperGuide.md

```diff
@@ -376,6 +376,95 @@ when used, instead of an invalid or unexpected entry.
             when the system is running other process concurrently.
         * Handling of behaviour for specific date types in terms of comparing dates may require more work.
 
+### Interaction record feature
+
+#### Implementation
+
+Each `Person` object contains its own list of `ContactedInfo` objects. These objects are stored in a List in each `Person` object.
```

Comment:
These objects are stored in a `List` in each `Person` or These objects are stored in a list in each `Person`?
`List` vs list